### PR TITLE
add support for aarch64-linux

### PR DESCRIPTION
**Pull request by flavorjones**
See #47 for context.

@larskanis I'd love to get feedback on this.

The additional changes I needed to make downstream to Nokogiri to successfully build are on a branch at https://github.com/sparklemotion/nokogiri/pull/2332

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -16,6 +16,7 @@ namespace :build do
     ["x86_64-linux", "x86_64-redhat-linux"],
     ["x86_64-darwin", "x86_64-apple-darwin"],
     ["arm64-darwin", "aarch64-apple-darwin"],
+    ["arm64-linux", "aarch64-linux-gnu"],
```
Comment (larskanis) [inline Rakefile:19]: I think this should be `["aarch64-linux"`. I haven't seen `arm64-linux` so far and also your [Graviton output](https://github.com/sparklemotion/nokogiri/pull/2332#issuecomment-929548470) calls it `gem_platform: aarch64-linux`.

Comment (flavorjones) [reply Rakefile:19]: Ah, of course. Will fix.

Comment (flavorjones) [reply Rakefile]: Updated.

**Comment by larskanis** [comment]
Thank you @flavorjones for your work! I'll try to find some time to make a new release.

**Comment by tienle** [comment]
Any schedule to release this soon? @larskanis

**Pull request merged**